src/middlewares/middleware.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const errorHandler = (err, req, res, next) => {
+  let statusCode = 500;
+
+  if (err.message === 'Not found') {
+    statusCode = 404;
+  } else if (err.message === 'Bad request') {
+    statusCode = 400;
+  }
```

Comment:
To set type depends on the message is not flexible, and in you can you have magic values. 
A simple example of an error handler https://github.com/BudnikOleksii/rick-n-morty-legacy-/blob/main/server/src/middlewares/error-handler.js
https://github.com/BudnikOleksii/rick-n-morty-legacy-/blob/main/server/src/utils/errors/api-errors.js

Reply:
Thank you very much, that's helpful)